src/test/javascript/spec/component/programming-assessment/code-editor-tutor-assessment-container.component.spec.ts

```diff
@@ -276,6 +276,42 @@ describe('CodeEditorTutorAssessmentContainerComponent', () => {
         flush();
     }));
 
+    it('should be able to override directly after submitting', () => {
+        jest.spyOn(comp, 'handleSaveOrSubmit').mockReturnValue(Promise.resolve());
+
+        const exercise = new ProgrammingExercise(undefined, undefined);
+        exercise.isAtLeastInstructor = true;
+        exercise.dueDate = Date.now();
```

Comment:
probably should be
```suggestion
        exercise.dueDate = dayjs();
```
or
```suggestion
        exercise.dueDate = dayjs.dayjs();
```
depending on how the library is imported